artifactory/commands/buildinfo/adddependencies.go

```diff
@@ -151,7 +151,7 @@ func prepareArtifactoryParams(specFile spec.File) (*specutils.ArtifactoryCommonP
 func getDependenciesBySpecFileParams(addDepsParams *specutils.ArtifactoryCommonParams) ([]string, error) {
 	addDepsParams.SetPattern(clientutils.ReplaceTildeWithUserHome(addDepsParams.GetPattern()))
 	// Save parentheses index in pattern, witch have corresponding placeholder.
-	rootPath, err := fspatterns.GetRootPath(addDepsParams.GetPattern(), addDepsParams.GetTarget(), addDepsParams.IsRegexp(), false)
+	rootPath, err := fspatterns.GetRootPath(addDepsParams.GetPattern(), addDepsParams.GetTarget(), addDepsParams.IsRegexp(), addDepsParams.IsAnt(),false)
```

Comment:
Notice that the total number of args can be reduced, ny just sending ```addDepsParams``` instead of the struct's properties separately.
There's also a missing white-spca before the last argument:
... ,false) --> ... , false)

Reply:
fspatterns.GetRootPath(..) is being called from artifactory (with ArtifactoryCommonParams variable) and from bintray (with UploadParams variable).
I changed it to be "addDepsParams.GetPatternType()" instead of passing regexp and ant for now.